app/routes/meetings/MeetingDetailRoute.tsx

```diff
@@ -63,12 +70,16 @@ const mapStateToProps = (state, props) => {
   const showAnswer = Boolean(
     meetingsToken.response === 'SUCCESS' && action && token
   );
+  const emojis = selectEmojis(state);
   return {
     meetingsToken,
     user: props.currentUser,
     showAnswer,
     meeting,
     currentUser,
+    emojis,
+    fetchingEmojis: state.emojis.fetching,
+    fetching: state.meetings.fetching,
```

Comment:
This seems to be unused.

Reply:
@jonasdeluna You removed `fetchingEmojis`, but that's actually used. I was only referring to `fetching`. Sorry for not being more specific 😅